framework/atoms/social-icons/social-icons.html

```diff
@@ -0,0 +1,172 @@
+<div style="float: left;">
+  <ul class="menu nav">
```

Comment:
I agree with you that the floating correction is ok to stay here instead of making a template for only 1 float